src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -66,7 +66,11 @@ case class BaggedSingleResult(
     * @return uncertainty of each prediction
     */
   override def getUncertainty(): Option[Seq[Any]] = Some(Seq(scalarUncertainty))
-  private lazy val scalarUncertainty = Math.sqrt(singleScores.sum) * rescale
+  private lazy val scalarUncertainty = if (useJackknife) {
+    Math.sqrt(singleScores.sum * Math.pow(rescale, 2.0) + Math.pow(bias.getOrElse(0.0), 2.0))
```

Comment:
This is the primary bugfix.  As in `Bagger`, we're applying the scale factor for the jackknife component of the uncertainty only.

Reply:
@sparadiso @bfolie @latture

Reply:
We were actually just discussing a similar change--this brings BaggedSingleResult and BaggedMultiResult into alignment, in that they both follow the same logic for calculating uncertainty. However, there's still the question of why the uncertainty is 0. If useJacknife is false, then the uncertainty is equal to the bias, but the bias learner is predicting 0. Why?

Reply:
@maxhutch Can we add a test when `rescale != 1.0`? I don't think this that the equation to calculate uncertainty in the multi-result will give the same values in that case.